File: 2024/test/day01_test.fs
module Day01Tests

open Xunit
open Day01

let input =
    "3   4
4   3
2   5
1   3
3   9
3   3"

[<Fact>]
let ``test part 1`` () = runPartOne input =! 11

[<Fact>]
let ``test part 2`` () = runPartTwo input =! 31

File: 2024/test/day01.test.ts
import { assertEquals } from "jsr:@std/assert";
import { runPartOne, runPartTwo } from "../src/day01.ts";

const input = `3   4
4   3
2   5
1   3
3   9
3   3`;

Deno.test("test day 01 part 1", () => {
  assertEquals(runPartOne(input), 11);
});

Deno.test("test day 01 part 2", () => {
  assertEquals(runPartTwo(input), 31);
});
